starters/gatsby-starter-catalyst-bery/src/gatsby-plugin-theme-ui/index.js

```diff
@@ -69,9 +70,10 @@ export default merge(BaseTheme, {
   },
   styles: {
     a: {
-      color: "link",
```

Comment:
I made this change due to a11y - link colors in text can be really finicky. The primary color unfortunately does not have enough contrast against the background and also against the surrounding text. It is actually really tough to meet these accessibility guidelines.

You can check out some detail on that here: https://www.w3.org/WAI/WCAG21/working-examples/link-contrast/